changelogs/changelog.yaml

```diff
@@ -2692,7 +2692,7 @@ releases:
       - iam_user_info - the ``path`` parameter has been renamed ``path_prefix`` for
         consistency with other IAM modules, ``path`` remains as an alias. No change
         to playbooks is required (https://github.com/ansible-collections/amazon.aws/pull/1933).
-    release_summary: This release includes new features and a bugfix.
+      release_summary: This release includes new features and a bugfix.
```

Comment:
is this expected? Is the YAML file still valid with this update?

Reply:
It was not valid before, and that release summary section was getting removed from the rst file when running `antsibull-changelog release` or `antsibull-changelog generate`.

Reply:
ok got it